integration/src/integration.ts

```diff
@@ -136,6 +137,14 @@ export function integration(suite: WalletSuite): void {
       thorchainTests(() => ({ wallet, info }));
     });
 
+    describe("mayachainTests", () => {
```

Comment:
This doesn't follow the naming convention of tests in this file. Shouldn't this be `MayaChainWallet` ?